packages/api/src/platforms/vtex/resolvers/product.ts

```diff
@@ -56,16 +77,26 @@ export const StoreProduct: Record<string, Resolver<Root>> & {
   brand: ({ isVariantOf: { brand } }) => ({ name: brand }),
   unitMultiplier: ({ unitMultiplier }) => unitMultiplier,
   breadcrumbList: ({
-    isVariantOf: { categories, productName, linkText },
+    isVariantOf: {
+      categories,
+      productName,
+      linkText,
+      categoryId,
+      categoriesIds,
+    },
     itemId,
   }) => {
+    const mainTreeIndex = findMainTreeIndex(categoriesIds, categoryId)
+    const mainTree = categories[mainTreeIndex]
+    const splittedCategories = removeTrailingSlashes(mainTree).split('/')
+
     return {
       itemListElement: [
```

Comment:
@lariciamota do you think we should add this notice somewhere?

> If IS starts providing the list without similar categories, we should update the code since we'll have direct access and won't need this logic.

Reply:
Sure! I'll add it